res/database/Default/database_programmes.xml

```diff
@@ -11514,10 +11512,9012 @@ Dream scenes and sentiments warp the storyline.</en>
+			<staff>
+				<member index="0" function="1">Stukov_BradfordMay</member>
+				<member index="1" function="2">a2be83c0-63a6-4ddd-9b60-953662a188f6</member>
+			</staff>
+			<groups target_groups="0" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="USA" distribution="2" maingenre="16" subgenre="17" flags="0" blocks="3" price_mod="1" />
+			<releaseTime year="1997" day="2" />
+			<ratings critics="20" speed="35" outcome="27" />
+		</programme>
+		<programme id="Stukov_Filme_Hobbit" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0077687" creator="9584" created_by="Stukov">
+			<title>
+				<de>Der Halbling</de>
+				<en>The Halfling</en>
+			</title>
+			<title_original>
+				<de>Der Hobbit</de>
+				<en>The Hobbit</en>
+			</title_original>
+			<description>
+				<de>Ein häuslicher Hobbit in Mittelerde wird überredet, sich einer Gruppe von Halblingen anzuschließen, um ihren Schatz aus den Fängen eines Drachens zu befreien.</de>
+				<en>A homebody hobbit in Middle Earth gets talked into joining a quest with a group of halflings to recover their treasure from a dragon.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">Stukov_JulesBass</member>
+			</staff>
+			<groups target_groups="3" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="USA/J" year="1977" distribution="2" maingenre="3" subgenre="10,9" flags="2" blocks="2" price_mod="0.50" />
+			<ratings critics="65" speed="61" outcome="38" />
+		</programme>
+		<programme id="Stukov_Filme_Der Herr der Ringe" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0077869" creator="9584" created_by="Stukov" comment="targetgroupattractivity: More adult-oriented than usual cartoons, i.e. fantasy. TODO: Clarify: Does this override the definition for the genre in programmedatamods.xml? Or are these parameters applied after the calculation?">
+			<title>
+				<de>Meister Der Dinge</de>
+				<en>The Master Of Things</en>
+			</title>
+			<title_original>
+				<de>Der Herr der Ringe</de>
+				<en>The Lord of the Rings</en>
+			</title_original>
+			<description>
+				<de>Die Gefährten des Rings begeben sich auf eine Reise, um den einen Ring zu zerstören und Saurons Herrschaft über Mittelerde zu beenden.</de>
+				<en>The Fellowship of the Ring embarks on a journey to destroy the One Ring and end Sauron's reign over Middle Earth.</en>
+			</description>
+			<staff>
+				<member index="0" function="1">Stukov_RalphBakshi</member>
+			</staff>
+			<groups target_groups="3" pro_pressure_groups="0" contra_pressure_groups="0" />
+			<data country="USA/GB" year="1978" distribution="1" maingenre="3" subgenre="1,10" flags="2" blocks="3" price_mod="0.50" />
+			<targetgroupattractivity children="1.5" teenagers_male="1.5" teenagers_female="0.5" housewives_male="0" housewives_female="0.2" employees_male="0.3" employees_female="0" unemployed_male="0.3" unemployed_female="0" managers="0" pensioners_male="-1.5" pensioners_female="-2" />
```

Comment:
Base value is 1, range 0 to 2; negative values will be turned into 0.
children 1.5 contradicts "adult-oriented".

Reply:
My original understanding was it would replace the original genre definition, so I thought it would lower children attractivity from 2.0 to 1.5 (children still like animations of course).

But wow, this works not at all like I thought. There are a lot of negative values used in `programmedatamods.xml` - so those are basically wrong?

I'll adapt all these values in the file according to my new understanding.

Reply:
Negative values will simply be replaced by 0, so it makes no difference if you write -0.3 or -2.
The foremost reason why I want the values changed accordingly is because existing database entries are likely to be used as bases for further additions, so "errors" easily propagate.